projects/chat-lib/src/lib/chat-window/chat-window.component.ts

```diff
@@ -1,32 +1,32 @@
 import { Component, OnInit, Input, AfterViewInit } from '@angular/core';
-
+import { InputValues } from '../../model/input'
 @Component({
   selector: 'lib-chat-window',
   templateUrl: './chat-window.component.html',
   styleUrls: ['./chat-window.component.scss']
 })
 export class ChatWindowComponent implements OnInit {
 
-  @Input() collapsed = true;
-  @Input() did: string;
-  @Input() userId: string;
-  @Input() channel: string;
-  @Input() appId: string;
-  @Input() title: string;
-  @Input() imageUrl: string;
-  @Input() chatbotUrl: string;
-  @Input() context: string;
-
+  @Input() inputValues: InputValues
   constructor() { }
-
+  imageClass = 'defaultImg';
+  textbox = 'chatboxDefault'
   ngOnInit() {
+    this.inputValues.collapsed = true;
+    this.inputValues.title ? this.inputValues.title : this.inputValues.title = "default"
+    if (this.inputValues.imageUrl) {
+      this.inputValues.imageUrl = this.inputValues.imageUrl;
+      this.imageClass = 'chatboxImg'
```

Comment:
please use the class in HTML using the ngClass and condition and also use a ? : operator